### PR TITLE
Feat/210/GitHub weekly participation by project

### PR DESCRIPTION
**Pull request by martinrobled0**
# What does this PR do?
Get the weekly participation by project statistics of github.

# How should this be manually tested?

1. make sure at least one project has its repository url registered
2. Run the _get-gitprojects-activity_ script
3. Go to Git hub info page of the project
4. Check the statistics /github-info

# Any background context you want to provide?

# What are the relevant tickets?
#210 

# Screenshots
image

image

# Questions

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

**Review by CarlosMilano — COMMENTED** [review]
Everything else looks good!

File: app/models/githubactivity.server.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { prisma } from "../db.server";
+import type { PrismaClient } from "@prisma/client";
+
+interface gitHubActivityChartType {
+    count: number,
+    typeEvent: string,
+}
+
+export async function saveACtivity(
```
Comment (AxelUZ) [inline app/models/githubactivity.server.ts:9]: Maybe we could change here to "saveActivity" to follow the camelCase style.

**Review by CarlosMilano — APPROVED** [review]
LGTM!

**Review by AxelUZ — APPROVED** [review]
It looks good to me 👍

**Pull request merged**